src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -118,9 +170,14 @@ rules:
 Built-in Workflow State Triggers
 --------------------------------
 
-These can be used instead of the older workflow state polling tasks described
-in :ref:`WorkflowStatePolling` for inter-workflow triggering - i.e. to trigger local
-tasks off of remote task statuses or messages in other workflows.
+Workflow-state xtriggers succeed when a target task in another workflow achieves
+a given state or output.
```

Comment:
I think it's worth separating the general introduction from the explanation of why the terminology seems backwards (the target is the upstream dependency).

```suggestion
Workflow-state xtriggers succeed when a task in another workflow achieves
a given state or output.

The task in the other workflow is called the target, because it is the target of
the xtrigger.
```

Reply:
>  from the explanation of why the terminology seems backwards 

I'm not sure I understand this comment. Did you omit some punctuation?

I prefer the original as I'm using "target" as merely an adjective. It's just saying we are targeting a particular task in another workflow - that's the task whose state we are interested. To me that's just normal descriptive English.  Your version sounds more like formalising "target" as special terminology, which wasn't my intention.